src/main/java/flaxbeard/steamcraft/tile/TileEntitySteamCharger.java

```diff
@@ -284,4 +355,4 @@ public float getSteamInItem() {
         return 0.0f;
     }
 
-}
+}
```

Comment:
New line here too would be nice (this one was probably Flaxbeard's fault though).